### PR TITLE
chore: add issue template to repo

### PR DESCRIPTION
**Pull request by lloydstubber**
After going through multiple issues and speaking with a few campers who have recently opened issues I’ve come up with an issue template. My main concern which @ActuallyACat brought up was to keep it brief or it’ll get ignored, so hoping this is a good start.

Could potentially trim down the labelling part and we handle that ourselves. WDYT?

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+### If it's a discussion, please start one [here](https://github.com/cultureamp/kaizen-design-system/discussions).
+
+- [ ] Before opening this issue, I've searched open and closed issues for a fix.
+
+## Expected Behavior
+
+
+## Actual Behavior (please provide screenshots)
```
Comment (jasononeil) [inline .github/issue_template.md:8]: Some of our issues are code only, should we clarify that? 

```suggestion
## Actual Behavior (please provide screenshots if it's a visual issue)
```

File: .github/issue_template.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## Steps to Reproduce the Problem
+
+1.
+2.
+3.
+
+## Specifications (please add any other notable libs/details)
+
+- Kaizen version:
```
Comment (jasononeil) [inline .github/issue_template.md:19]: I like including this, should go a long way to identifying if an issue might have been subtly fixed by a recent release

File: .github/issue_template.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+- Repo:
```
Comment (jasononeil) [inline .github/issue_template.md:20]: When I read this, my first though was "won't the repo always be kaizen-design-system?"

But I'm guessing this is about a bug where the problem might have been encountered? Is there a way to capture that in the name?

In our team we often talk about "consuming repos" but I am not sure that language would be obvious to people outside our team

File: .github/issue_template.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+## Labelling
+
+1. If related to a component, please add a label like so `component:mycomponent`.
+2. Then label with `bug`, `enhancement`/`feature`\* or `documentation`.
+
+**\* A feature is a new component/functionality and an enhancement is an improvement on existing component/functionality.**
```
Comment (jasononeil) [inline .github/issue_template.md:27]: Is functionality a plural? If I read "a new functionality" it sounds wrong. I'm tempted to reword this to avoid that :)

```suggestion
**\* A feature is a new component or piece of functionality and an enhancement is an improvement to an existing component or piece of functionality.**
```

Comment (lloydstubber) [reply .github/issue_template.md:27]: Good catch, moved it around and missed that.

File: .github/issue_template.md (hunk 3 above)
Comment (lloydstubber) [reply .github/issue_template.md:20]: I was more going for "what repo you're working in?" for our context. Like `survey-summary-frontend` or maybe that's not useful to us?

File: .github/issue_template.md (hunk 1 above)
Comment (lloydstubber) [reply .github/issue_template.md:8]: Yep I like that.

File: .github/issue_template.md (hunk 2 above)
Comment (lloydstubber) [reply .github/issue_template.md:19]: Totally. Was also wondering if it's worth adding like a `Kaizen component version:` if it's like `"@kaizen/brand": "^1.1.1",` but maybe that's enough?

File: .github/issue_template.md (hunk 3 above)
Comment (ActuallyACat) [reply .github/issue_template.md:20]: If it's an issue in production, the repo and/or location in the product is useful. If it's an issue someone has encountered in work that's in progress, a branch is useful. It would be hard to capture these things succinctly - I wouldn't be opposed to excluding this for brevity and let whoever opened the issue to provide whatever information they think is useful.

**Review by ActuallyACat — APPROVED** [review]
Looks good 👍🏻
